### DISCUSSION TITLE
微軟第二代聊天機器人

### DISCUSSION THREAD

**Discussion by 109302003** [1101-GS4719-C/Python-Community-GS4719-C#314] [General]
[https://innoservice.org/12131/%e6%a1%88%e4%be%8b-%e5%be%ae%e8%bb%9f%e5%86%8d%e5%87%ba%e8%81%8a%e5%a4%a9%e6%a9%9f%e5%99%a8%e4%ba%ba-zo%ef%bc%8c%e9%80%99%e6%ac%a1%e4%b8%8d%e8%81%8a%e6%95%8f%e6%84%9f%e8%a9%b1%e9%a1%8c/]() 

經過第一代Tay的慘痛經歷，第二代聊天機器人被限制不談論敏感話題，我覺得限制敏感話題與遵從特定立場及想法有非常大的差別。

**Comment by chun1203** [comment]
這個連結好像用錯了 沒辦法看QQ
